doc/articles/features/working-with-xaml-hot-reload.md

```diff
@@ -57,19 +64,19 @@ WebAssembly is currently providing both full and partial Hot Reload support, dep
 
 Mobile targets are currently using a limited version of XAML Hot Reload and do not support C# Hot Reload until [this dotnet runtime](https://github.com/dotnet/runtime/issues/93860) issue is fixed.
 
-- In Visual Studio, the "Hot Reload on File Save" feature must be disabled to avoid crashing the app
-- In both VS and VS Code, C# Hot Reload is not supported
-- x:Bind hot reload is limited to simple expressions and events
+> [!NOTE]
+> Help us by giving it a thumbs up to help prioritize it in the .NET team backlog!
+
+- In Visual Studio, the "Hot Reload on File Save" feature must be disabled to avoid crashing the app. You can find this feature by clicking on the down arrow next to the red flame in the Visual Studio tool bar.
```

Comment:
```suggestion
- In Visual Studio, the "Hot Reload on File Save" feature must be disabled to avoid crashing the app. You can find this feature by clicking on the down arrow next to the red flame in the Visual Studio toolbar.
```